### PR TITLE
[Research] make the highlighted poi and the yellow flag (when creating a report) centered at the map rather than screen.

### PR DESCRIPTION
**Pull request by hsinlun99**
**NEED MERGE https://github.com/CAMPUS-NYCU/smart-campus/pull/66 #67 FIRST**
# Description
- highlighted poi
image
- yellow flag
image

# Changes

# Notes

# Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] Any changes to strings have been published to our translation tool
- [x] I conducted basic QA to assure all features are working
- [x] I requested code review from other team members

# Resolved Issues
https://github.com/CAMPUS-NYCU/smart-campus/issues/68

### CODE REVIEW

File: src/components/Drawer/AddReportDrawer/index.tsx [hunk 1]
```diff
@@ -72,11 +72,16 @@ const AddReportDrawer: React.FC = () => {
...
     }
 
-    const latlng = { latitude: center.lat(), longitude: center.lng() };
+    const latlng = {
+      latitude: north - (north - south) / 4,
+      longitude: center.lng(),
+    };
     dispatch(updateAddReportData({ latlng }));
   };
   const handleDrawerConfirm = () => {
```
Comment (linyc0817) [inline src/components/Drawer/AddReportDrawer/index.tsx:87]:
```suggestion
  };
  
  const handleDrawerConfirm = () => {
```

Add a space here.

File: src/components/Drawer/AddReportDrawer/index.tsx [hunk 2]
```diff
@@ -72,11 +72,16 @@ const AddReportDrawer: React.FC = () => {
...
-    if (!center) {
+    const north = maps.getBounds()?.getNorthEast().lat();
+    const south = maps.getBounds()?.getSouthWest().lat();
+    if (!center || !north || !south) {
       throw new Error("LatLng not found");
     }
 
-    const latlng = { latitude: center.lat(), longitude: center.lng() };
+    const latlng = {
+      latitude: north - (north - south) / 4,
```
Comment (linyc0817) [inline src/components/Drawer/AddReportDrawer/index.tsx:82]:
```suggestion
      latitude: south + (north - south) * 0.75,
```
Will this be better to understand? Is no just ignore it !

**Review by linyc0817 — CHANGES_REQUESTED** [review]
Well done, only some format need change!

File: src/components/Drawer/AddReportDrawer/index.tsx (hunk 2 above)
Comment (hsinlun99) [reply src/components/Drawer/AddReportDrawer/index.tsx:82]: 我覺得...都行？
不然就改成你那樣好了哈哈哈

**Pull request merged**